pyiceberg/catalog/rest.py

```diff
@@ -290,7 +293,14 @@ def url(self, endpoint: str, prefixed: bool = True, **kwargs: Any) -> str:
 
     @property
     def auth_url(self) -> str:
-        if url := self.properties.get(AUTH_URL):
+        if self.properties.get(DEPRECATED_AUTH_URL):
```

Comment:
I think the logic is a bit more elaborate, see: https://github.com/apache/iceberg/pull/10603/files#diff-86450612dbe323d6d06cbc3846aa1913f042eaedadc0ca027c36bfbe08d3a46cR196

Reply:
The initial purpose of this PR was not to warn about the `/oauth/tokens` endpoint deprecation, it was only to fix the configuration property, but I can add this.